src/site/content/en/lighthouse-accessibility/aria-name/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title: "ARIA items do not have accessible names"
+description: |
+  Learn how to improve your web page's accessibility by making sure that
+  assistive technology users can access the names of ARIA items.
+date: 2020-12-03
+web_lighthouse:
+  - aria-command-name
+  - aria-input-field-name
+  - aria-meter-name
+  - aria-progressbar-name
+  - aria-toggle-field-name
+  - aria-tooltip-name
+  - aria-treeitem-name
+---
+
+{% include 'content/lighthouse-accessibility/about-aria.njk' %}
+
+{% include 'content/lighthouse-accessibility/accessible-names.njk' %}
+
+## How Lighthouse identifies ARIA items without accessible names
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/)
+flags custom ARIA items whose names
+aren't accessible to assistive technologies:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="aria-toggle-field-name.png"
+    alt="Lighthouse audit showing custom toggle elements without accessible names">
+</figure>
+
+There are 7 audits that check for accessible names, each one covers a different
+set of [ARIA roles](https://www.w3.org/TR/wai-aria-practices-1.1/#aria_ex). Elements
+that have any of the following ARIA roles but don't have accessible names will cause this audit to fail:
+- `aria-command-name` - `button`, `link`, `menuitem`
+- `aria-input-field-name` - `combobox`, `listbox`, `searchbox`, `slider`, `spinbutton`, `textbox`
+- `aria-meter-name` - `meter`
+- `aria-progressbar-name` - `progressbar`
+- `aria-toggle-field-name` - `checkbox`, `menu`, `menuitemcheckbox`, `menuitemradio`, `radio`, `radiogroup`, `switch`
+- `aria-tooltip-name` - `tooltip`
+- `aria-treeitem-name` - `treeitem`
+
+{% include 'content/lighthouse-accessibility/use-native.njk' %}
+
+{% include 'content/lighthouse-accessibility/scoring.njk' %}
+
+## Example 1: How to add accessible names to your custom ARIA toggle fields
+
+### Option 1: Add inner text to the element
+
+The easiest way to provide an accessible name for most elements
+is to include text content inside the element.
+
+For example, this custom checkbox will be announced as "Newspaper"
+to assistive technology users:
+
+```html
+<div id="checkbox1" role="checkbox">Newspaper</div>
+```
+
+Using the ["clip pattern"](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/) you can hide the inner text on screen, but still have it announced by assistive technology. This can be especially handy if you translate your pages for localization.
```

Comment:
```suggestion
Using the [clip pattern](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/) you can hide the inner text on screen, but still have it announced by assistive technology. This can be especially handy if you translate your pages for localization.
```